File: Accelerometer Sensor/app/src/main/java/com/example/sensor/MainActivity.java
package com.example.sensor;

import androidx.appcompat.app.AppCompatActivity;

import android.hardware.Sensor;
import android.hardware.SensorEvent;
import android.hardware.SensorEventListener;
import android.hardware.SensorManager;
import android.os.Bundle;
import android.widget.TextView;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity implements SensorEventListener {

    TextView txtValues;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        txtValues=findViewById(R.id.txtValues);

        SensorManager sensorManager= (SensorManager) getSystemService(SENSOR_SERVICE);

        if (sensorManager!=null){
            Sensor acceleroSensor = sensorManager.getDefaultSensor(Sensor.TYPE_ACCELEROMETER);
            if (acceleroSensor!=null){
                sensorManager.registerListener(this, acceleroSensor,SensorManager.SENSOR_DELAY_NORMAL);
            }
        }else{
            Toast.makeText(this, "Sensor Service not detected", Toast.LENGTH_SHORT).show();
        }

    }

    @Override
    public void onSensorChanged(SensorEvent sensorEvent) {
        if (sensorEvent.sensor.getType()==Sensor.TYPE_ACCELEROMETER){
            txtValues.setText("X :"+sensorEvent.values[0]+", Y: "+sensorEvent.values[1]+", z: "+sensorEvent.values[2]);
        }
    }

    @Override
    public void onAccuracyChanged(Sensor sensor, int i) {

    }
}

File: Calculator/app/src/main/java/com/example/calculator/MainActivity.kt
package com.example.calculator

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Toast
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btnAdd.setOnClickListener {
            if (edtNo1.text.toString()!="" && edtNo2.text.toString()!=""){
                val no1 = edtNo1.text.toString().toInt()
                val no2 = edtNo2.text.toString().toInt()
                val sum = no1 + no2

                Toast.makeText(this, "The Sum is $sum", Toast.LENGTH_SHORT).show()

            }else{
                Toast.makeText(this, "Please Fill All the Blanks", Toast.LENGTH_SHORT).show()
            }
        }
        btnSub.setOnClickListener {
            if (edtNo1.text.toString()!="" && edtNo2.text.toString()!=""){
                val no1 = edtNo1.text.toString().toInt()
                val no2 = edtNo2.text.toString().toInt()

                val sub : Int = if (no1>no2){
                    no1-no2
                }else{
                    no2 - no1
                }

                Toast.makeText(this, "The Subtract is $sub", Toast.LENGTH_SHORT).show()

            }else{
                Toast.makeText(this, "Please Fill All the Blanks", Toast.LENGTH_SHORT).show()
            }
        }
        btnMul.setOnClickListener {
            if (edtNo1.text.toString()!="" && edtNo2.text.toString()!=""){
                val no1 = edtNo1.text.toString().toInt()
                val no2 = edtNo2.text.toString().toInt()
                val mul = no1 * no2

                Toast.makeText(this, "The Multiplication is $mul", Toast.LENGTH_SHORT).show()

            }else{
                Toast.makeText(this, "Please Fill All the Blanks", Toast.LENGTH_SHORT).show()
            }
        }
        btnDiv.setOnClickListener {
            if (edtNo1.text.toString()!="" && edtNo2.text.toString()!=""){
                val no1 = edtNo1.text.toString().toInt()
                val no2 = edtNo2.text.toString().toInt()
                val div : Int
                if (no2==0){
                    div = 0
                    Toast.makeText(this, "A number can't be divided by zero as it produces infinite solutions", Toast.LENGTH_LONG).show()
                }else{
                     div = (no1 / no2)
                }

                Toast.makeText(this, "The Division is $div", Toast.LENGTH_SHORT).show()

            }else{
                Toast.makeText(this, "Please Fill All the Blanks", Toast.LENGTH_SHORT).show()
            }
        }

    }
}